packages/shared/src/components/RichTextEditor/plugins/ButtonPlugin/ButtonMarkupElement/ButtonMarkupElementNode.tsx

```diff
@@ -0,0 +1,94 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useRichTextEditorContext } from '@frontify/fondue';
+import { ButtonStyles } from '@frontify/fondue/dist/components/RichTextEditor/Plugins/TextStylePlugin/TextStyles';
+import { DesignTokens } from '@frontify/fondue/dist/components/RichTextEditor/types';
+import { HTMLPropsAs, PlateRenderElementProps, Value, useElementProps } from '@udecode/plate';
```

Comment:
Please make a ticket for next sprint to remove as much as possible imports from plate and only rely on fondue.

Reply:
Done: https://app.clickup.com/t/8677uxbw6